xfce/libxfce4util/Portfile

```diff
@@ -3,11 +3,11 @@
 PortSystem 1.0
 
 name            libxfce4util
-version         4.16.0
+version         4.18.2
```

Comment:
Why not upgrade to the current version?

Reply:
As far as I understand, 4.19.x versions are development versions for what will ultimately become the 4.20.x series.